pkgs/tools/misc/mslink/default.nix

```diff
@@ -17,6 +17,9 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
+    if [[ "$(uname)" == "Darwin" ]]; then
+      mv mslink.exe mslink
```

Comment:
Would it hurt to do this on all platforms?

Reply:
it checks OS != Linux -> .exe, so this is not required there